commands/runner.go

```diff
@@ -54,8 +54,11 @@ func (r *Runner) All() map[string]*Command {
 	return r.commands
 }
 
-func (r *Runner) Use(command *Command) {
+func (r *Runner) Use(command *Command, aliases ...string) {
```

Comment:
How about adding a alias field to `Command` which would make it more modular:

```go
type Command struct {
   Aliases []string
}
```

In `Use`, we could add all aliases:

```go
for _, a := range command.Aliases {
   r.commands[a] = command
}
```